4-spring-web-app/1-Authentication/sign-in/src/main/resources/application.yml

```diff
@@ -1,17 +1,22 @@
-azure:
-  activedirectory:
-    # Specifies your Active Directory ID:
-    tenant-id: Enter_Your_Tenant_ID_Here
-    # Specifies your App Registration's Application ID:
-    client-id: Enter_Your_Client_ID_Here
-    # Specifies your App Registration's secret key:
-    client-secret: Enter_Your_Client_Secret_Here
-    # Specifies the post-log-out-redirect-uri, where to return your app after logout.
-    post-logout-redirect-uri: http://localhost:8080
-    # Specifies the Microsoft Graph scopes that your app needs access to - not required in this app.
-    # authorization-clients:
-    #   graph:
-    #     scopes: https://graph.microsoft.com/User.Read
+spring:
+  cloud:
+    azure:
+      active-directory:
+        enabled: true
+        profile:
+          # Specifies your Active Directory ID:
+          tenant-id: Enter_Your_Tenant_ID_Here
+        credential:  
+          # Specifies your App Registration's Application ID:
+          client-id: Enter_Your_Client_ID_Here
```

Comment:
Not sure I get this config file

- is "enabled" really needed?
- do we need "profile" and "credential"? I would keep them flat. client-id is not part of the "credential" in any case.

Reply:
Ack, thank you. Please add a link to this doc in the yaml.

Reply:
Added the link. Thanks